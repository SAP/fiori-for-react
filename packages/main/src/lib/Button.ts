import { Button, ButtonPropTypes } from '../webComponents/Button';

export { Button, ButtonPropTypes };
