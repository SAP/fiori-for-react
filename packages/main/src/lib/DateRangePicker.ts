import { DateRangePicker } from '../webComponents/DateRangePicker';

export { DateRangePicker };
