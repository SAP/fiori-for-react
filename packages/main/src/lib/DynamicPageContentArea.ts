import { DynamicPageContentArea } from '../components/DynamicPageContentArea';

export { DynamicPageContentArea };
