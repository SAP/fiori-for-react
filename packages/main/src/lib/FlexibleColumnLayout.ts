import { FlexibleColumnLayout } from '../webComponents/FlexibleColumnLayout';

export { FlexibleColumnLayout };
