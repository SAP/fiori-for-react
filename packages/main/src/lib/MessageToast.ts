import { MessageToast } from '../components/MessageToast';

export { MessageToast };

// Required for React lazy
export default MessageToast;
