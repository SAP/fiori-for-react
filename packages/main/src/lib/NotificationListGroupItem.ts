import { NotificationListGroupItem } from '../webComponents/NotificationListGroupItem';

export { NotificationListGroupItem };
