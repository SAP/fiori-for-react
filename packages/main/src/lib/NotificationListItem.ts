import { NotificationListItem } from '../webComponents/NotificationListItem';

export { NotificationListItem };
