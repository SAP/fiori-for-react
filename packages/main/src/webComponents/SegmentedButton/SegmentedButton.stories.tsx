import React from 'react';
import { SegmentedButton } from '@ui5/webcomponents-react/lib/SegmentedButton';
import { action } from '@storybook/addon-actions';
import {} from '@storybook/addon-knobs';
export default {
  title: 'UI5 Web Components / SegmentedButton',
  component: SegmentedButton
};

export const generatedDefaultStory = () => (
  <SegmentedButton onSelectionChange={action('onSelectionChange')}>Some Content</SegmentedButton>
);

generatedDefaultStory.story = {
  name: 'Generated default story'
};
