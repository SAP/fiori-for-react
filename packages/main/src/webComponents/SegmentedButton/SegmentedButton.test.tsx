import { mount } from 'enzyme';
import React from 'react';
import { SegmentedButton } from '@ui5/webcomponents-react/lib/SegmentedButton';

describe('SegmentedButton', () => {
  test('Basic Test (generated)', () => {
    const wrapper = mount(<SegmentedButton />);
    expect(wrapper.render()).toMatchSnapshot();
  });
});
