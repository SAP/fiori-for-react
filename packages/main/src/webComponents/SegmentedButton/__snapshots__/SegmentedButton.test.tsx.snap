// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SegmentedButton Basic Test (generated) 1`] = `
<DocumentFragment>
  <ui5-segmentedbutton />
</DocumentFragment>
`;
